tools/figma-inspector/shared/universals.d.ts

```diff
@@ -27,7 +27,7 @@ export interface EventTS {
 
 export interface EventTS {
     monitorVariableChanges: { enabled: boolean };
-    checkVariableChanges: {};
+    checkVariableChanges: Record<string, never>;
```

Comment:
Is this an unrelated change?

Reply:
No, I think it's okay. I just wonder what change in this PR triggered this. But it seems harmless.